test-engineering/load-tests/setup_k8.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+set -eu
+
+#Declare variables
+GCLOUD=$(which gcloud)
+SED=$(which sed)
+KUBECTL=$(which kubectl)
+GOOGLE_CLOUD_PROJECT=$(gcloud config get-value project)
+CLUSTER=merino-locust-load-test
+TARGET='https://stage.merino.nonprod.cloudops.mozgcp.net'
+SCOPE='https://www.googleapis.com/auth/cloud-platform'
+REGION=us-central1
+WORKER_COUNT=10
```

Comment:
I wonder if this should be 10 or 20. If we're using 2-core machines (n1-standard-2) and 10 workers (and `--min-nodes=10` and `--max-nodes=15`) there might be more room to push the cluster without any additional cost.
Although technically out of the scope of this PR I guess.

Reply:
I think we should atleast try with 20 and see what happens locally and if it works fine, we can definitely make the WORKER_COUNT = 20.